config/settings.yml

```diff
@@ -1686,3 +1686,12 @@ brd:
 
 xlsx_file_fetcher:
   github_access_token: ~
+
+vye:
+  ivr_key: '0615324c-5a8d-9fe1-3542-7b3b86df281f'
+  scrypt:
+    salt: '51463d6f-3b19-ae87-578a-7c4e3f1a5588'
```

Comment:
Are these values sensitive? As you can see in this file, sensitive values are replaced with `~`. They're then placed in AWS parameter store. See the section on **Steps to create a new secret value** on the platform website: https://depo-platform-documentation.scrollhelp.site/developer-docs/vets-api-on-eks#VetsAPIonEKS-SecretValues This will involve making a PRsto the vsp-infra-manifests repo like Trevor mentioned.

(note: if these values are sensitive, please change them since this repo is public-facing.)

Reply:
updated, please review